src/commands/support/abbreviations.ts

```diff
@@ -0,0 +1,23 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const abbreviations: CommandDefinition = {
+    name: 'abbreviations',
+    description: 'Links to most commonly used abbreviations',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const abbreviationsEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | Abbreviations',
```

Comment:
```suggestion
            title: 'FlyByWire Support | Abbreviations',
```

Change wording here as abbreviations are not exclusive to the a32nx. More of a 'support' thing.